s3upload.js

```diff
@@ -60,7 +60,7 @@ S3Upload.prototype.createCORSRequest = function(method, url) {
 S3Upload.prototype.executeOnSignedUrl = function(file, callback) {
     var xhr = new XMLHttpRequest();
     var fileName = file.name.replace(/\s+/g, "_");
-    xhr.open('GET', this.signingUrl + '?objectName=' + fileName, true);
+    xhr.open('GET', this.signingUrl + '?objectName=' + fileName + '&contentType=' + file.type, true);
```

Comment:
Any idea the browser-compatibility for detecting the mime type with `file.type`?  If not available in some browsers, maybe a backend fallback to `mime.lookup()` would be appropriate.